deploy/kind-deploy.sh

```diff
@@ -238,7 +239,11 @@ helm install minio minio/minio --namespace minio --set rootUser=minio,rootPasswo
 #Deploy NFS server provisioner
 echo -e "\n[*] Deploying NFS server provider ..."
 helm repo add --force-update nfs-ganesha-server-and-external-provisioner https://kubernetes-sigs.github.io/nfs-ganesha-server-and-external-provisioner/
-helm install nfs-server-provisioner nfs-ganesha-server-and-external-provisioner/nfs-server-provisioner
+if [ $ARCH == "arm64"]; then
```

Comment:
This should probably read:
```
if [ $ARCH == "arm64" ];
```
Otherwise bash complains:

bash: line 242: [: missing `]'